source/material_model/latent_heat.cc

```diff
@@ -517,8 +535,19 @@ namespace aspect
                   density_jumps.size() != transition_pressures.size() ||
                   transition_phases.size() != transition_pressures.size() ||
                   phase_prefactors.size() != transition_pressures.size()+1)
-                AssertThrow(false, ExcMessage("Error: At least one list that gives input parameters for the phase transitions has the wrong size. Currently checking against transition pressures. If phase transitions in terms of depth are desired, check to make sure Define transition by depth instead of pressure = true."));
+                AssertThrow(false, ExcMessage("Error: At least one list that gives input parameters for the phase "
+                                              "transitions has the wrong size. Currently checking against transition pressures. "
+                                              "If phase transitions in terms of depth inputs are desired, check to make sure "
+                                              "'Define transition by depth instead of pressure = true'."));
             }
+
+          // as the phase viscosity prefactors are all applied on top of each other, we have
+          // to scale them here so that they are relative gactors in comparison to the phase above
```

Comment:
I would rephrase this a bit (it took me some time to understand what you mean). What about: "as the phase viscosity prefactors are all applied multiplicatively on top of each other, we have to scale them here so that they are relative factors in comparison to the product of all phase above"